models/tokens/optimism/tokens_optimism_erc20.sql

```diff
@@ -11,25 +11,16 @@ END
 AS is_counted_in_tvl
 FROM (
 	SELECT
-	coalesce(c.contract_address, t.contract_address, b.contract_address) AS contract_address
+	c.contract_address
 	, coalesce(t.symbol,b.symbol) as symbol
 	, coalesce(t.decimals,b.decimals) as decimals
 	, coalesce(t.token_type,b.token_type) AS token_type
 	, coalesce(t.token_mapping_source, b.token_mapping_source) AS token_mapping_source
 
-	, ROW_NUMBER() OVER
```

Comment:
We should not remove this.

The row_number check is used to handle for failed bridge events where a user maps the wrong L1<>L2 token on the L1 bridge

Reply:
this assumes we always map the "correct" token